ci/build.ps1

```diff
@@ -61,21 +61,14 @@ if ($compiler -eq 'MINGW') {
   # in MSYS2, but we cannot build inside the MSYS2 shell.
   $cmakeGenerator = 'Ninja'
   $cmakeGeneratorArgs = '-v'
-  $mingwPackages = @('ninja', 'cmake', 'diffutils').ForEach({
-    "mingw-w64-$arch-$_"
-  })
 
+  if (Get-Command -Name "msys2" -ErrorAction SilentlyContinue) {
+    $MINGW_PREFIX = (& msys2 -c 'cygpath -w /') + "mingw$bits\bin"
```

Comment:
Should this be

```suggestion
    $MINGW_PREFIX = (& msys2 -c 'cygpath -w /') + "mingw$bits/bin"
```

Reply:
this should be fine
```console
$ /usr/bin/cygpath -w /
C:\msys64\
```

Reply:
the current problem is currently here:
https://github.com/neovim/neovim/runs/6291974592?check_suite_focus=true#step:5:74
which for some reason is unable to find `ninja`, but checking previous builds seems to indicate that this step is actually using `make`
https://github.com/neovim/neovim/runs/6308146076?check_suite_focus=true#step:4:189

this is probably related to the shared cache?

Reply:
so far it seems that ruby might be the culprit here, but moving that block further down should fix the issue